apps/console/src/features/applications/components/api-authorization/subscribed-api-resources.tsx

```diff
@@ -0,0 +1,506 @@
+        }
+    };
+
+    /**
+     * Resolves the header of the subscribed API Resources list.
+     */
+    const resolveSubscribedAPIResourcesListHeader = (subscribedAPIResource: AuthorizedAPIListItemInterface): 
+        ReactElement => (
+        <Header
+            as="h6"
+            className="header-with-icon"
+            data-componentId={ `${componentId}-heading` }
+        >
+            <Grid verticalAlign="middle">
+                <Grid.Row>
+                    <Grid.Column width={ 8 }>
+                        <Header.Content>
+                            <Grid verticalAlign="middle">
+                                <Grid.Row>
+                                    <Grid.Column width={ 16 }>
+                                        { subscribedAPIResource.displayName }
+                                    </Grid.Column>
+                                </Grid.Row>
+                            </Grid>
+                            <Header.Subheader className="mt-1">
+                                {
+                                    subscribedAPIResource.identifier
+                                        ? <Code withBackground>{ subscribedAPIResource.identifier }</Code>
+                                        : null
+                                }
+                            </Header.Subheader>
+                        </Header.Content>
+                    </Grid.Column>
+                    <Grid.Column width={ 8 }>
+                        {
+                            subscribedAPIResource.identifier
+                                ? (
+                                    <Header.Subheader>
+                                        { renderPolicyForAPIResource(subscribedAPIResource.policyId) }
+                                    </Header.Subheader>
+                                )
+                                : null
+                        }
+                    </Grid.Column>
+                </Grid.Row>
+            </Grid>
+        </Header>
+    );
+
+    /**
+     * Creates the actions of the subscribed API Resources list item. 
+     */
+    const createAccordionAction = (subscribedAPIResource: AuthorizedAPIListItemInterface):
+        SegmentedAccordionTitleActionInterface[] => {
+
+        return [ 
+            {
+                disabled: !isScopesAvailableForUpdate,
+                icon: "trash alternate",
+                onClick: () => setRemoveSubscribedAPIResource(subscribedAPIResource),
+                popoverText: t("extensions:develop.applications.edit.sections.apiAuthorization.sections" +
+                    ".apiSubscriptions.unsubscribeAPIResourcePopOver"),
+                type: "icon"
+            } 
+        ];
+    };
+
+    /**
+     * Resolves the content of the subscribed API Resources list item.
+     */
+    const resolveSubscribedAPIResourcesListContent = (subscribedAPIResource: AuthorizedAPIListItemInterface):
+        ReactElement => (
+
+        <>
```

Comment:
Remove this component as there is only one child

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/4300/commits/dc52d8ae5a27ca520a4ea0bf62bba8b5899ceb74